src/ImageService.Models.Dto/Responses/Message/ImageMessageResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ImageService.Models.Dto.Responses.Message
+{
+    public class ImageMessageResponse
```

Comment:
1) record
2) parentId?
3) mb better to use Id, not ImageId

Reply:
don't we want to create enum for all types of images? with enum we don't need create 4 same models

Reply:
parentId will be null here, because this response is for high-quality image request, so I think we dont need to add it

Reply:
about enum - i think, that image models will be different in the future but in enum they must have the same type, are they?